packages/webpack/src/loader.ts

```diff
@@ -85,7 +90,9 @@ export default async function vocabLoader(this: LoaderContext, source: string) {
     throw new Error(`Webpack didn't provide an async callback`);
   }
 
-  await initModuleLexer;
+  // this is necessary for the Web Assembly boot
+  await esModuleLexer.init;
+
```

Comment:
Could this be moved closer to the `findExportNames` call, just so it's obvious that they're related.